src/widget/left_click.rs

```diff
@@ -0,0 +1,49 @@
+use iced::advanced::widget::Tree;
+use iced::advanced::{mouse, Clipboard, Layout, Shell};
+use iced::event;
+
+use crate::widget::{decorate, Renderer};
+use crate::Element;
+
+pub fn left_click<'a, Message>(
+    content: impl Into<Element<'a, Message>>,
+    message: Message,
+) -> Element<'a, Message>
```

Comment:
We don't need this over `button` w/ bare styling?

Reply:
We have a `widget::transparent_button` now.